vispy/geometry/parametric.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Nicolas P. Rougier
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+from .normals import normals
```

Comment:
It's generally not a good idea to have the same thing done in multiple places in different ways (DRY), but given that the collections already need quite a bit of de-duplication, I can live with it.